src/views/LandingPage/SelectRole/RoleButton.vue

```diff
@@ -31,7 +31,8 @@ export default {
       if (this.computeRoleLowerCase === "customer") {
         this.$router.push(`/${this.computeRoleLowerCase}`)
       } else {
-        window.open(`https://frontend-dev.debio.network/${this.computeRoleLowerCase}`)
+        const baseUrl = process.env.VUE_APP_DEV_DEBIO_FRONTEND_URL
```

Comment:
I think it would be better to use the `store` for this. Probably putting this environment inside the `auth` store would be fine for now. 

But we would need to change that store name in the future because it doesn't contain only authentication data.